pkg/cloud/services/provider/provider_test.xgo

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
Is the file type `xgo` correct? 🤔 
I think we must rename this file to `provider_test.go`

Reply:
Nope.. It was too late in the work-day for me :sleeping:

I tried to write some unit tests for the function but failed to get it working with the `tokens.CreateResult` struct, which i couldn't mock. So if anyone has an idea how we could test this, feel free to teach me, otherwise I am going to remove this file.

Reply:
Ah, so this is just a place holder / template test file. I understand.
Isn't it better to keep the correct file type but skip the test early?

Reply:
I don't think the file is worth keeping around if we zero actual tests.

--- 

RE Mocking the `tokens.CreateResult`: We could copy the way `gophercloud` internally tests the `identity/v3/tokens` package, but that involves setting up a mock http server, so I am not sure if that is excessive.

Reply:
I removed the file in the latest push.